### ISSUE TITLE
Silencing happypack output

### ISSUE THREAD

**Issue by dnieh** [amireh/happypack#64]
Is it possible to silence HappyPack's output? I don't see anything in the readme pertaining to this functionality. My motivation behind this is when I output the `stats.json` file for profiling. HappyPack prepends this with the following:
```json
Happy[js]: Version: 2.1.1. Using cache? yes. Threads: 4 (shared pool)
Happy[js]: Loaded 226 entries from cache. (0 were stale)
Happy[js]: All set; signalling webpack to proceed.
{
  "errors": [],
  "warnings": [],
  "version": "1.13.1",
...
```

Although fairly easy to remove, if unmodified, this does lead to invalid json.

**Comment by amireh** [comment]
Hmm, did you try setting `verbose: false`? That should've taken care of it.

It is true that it's only been recently added to the docs (https://github.com/amireh/happypack#verbose-boolean) although that option was there from the beginning.

Let me know if that still doesn't work for you.

**Comment by dnieh** [comment]
Even with that set, I still get the following output:
```
Happy thread[2] is now open.
Happy thread[3] is now open.
Happy thread[1] is now open.
Happy thread[0] is now open.
Happy[js]: All set; signalling webpack to proceed.
```

**Comment by amireh** [comment]
Fixed in 2.2.1.
